apps/backend/src/config.ts

```
@@ -148,6 +148,12 @@ export function verifyConfig(): void {
 			"BRAVE_SEARCH_API_KEY, BRAVE_SEARCH_API_URL and BRAVE_SEARCH_MAX_RPS or STAAN_SEARCH_API_KEY, STAAN_SEARCH_API_URL and STAAN_SEARCH_MAX_RPS must be defined when FEATURE_FLAG_WEB_SEARCH_ALLOWED is true",
 		);
 	}
+
+	if (process.env.PORT.length > 0) {
+		console.log(`Port set via env, value: ${process.env.PORT}`);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)